FBSnapshotTestCase/FBSnapshotTestCase.h

```diff
@@ -8,6 +8,8 @@
  *
  */
 
+#import <FBSnapshotTestCase/FBSnapshotTestCaseArchitecture.h>
```

Comment:
I am not sure about the name in general and where should I break things. This is just for now to push the code.